data/json/monsters.json

```diff
@@ -2107,7 +2107,7 @@
     "id": "mon_shadow",
     "type": "MONSTER",
     "name": "shadow",
-    "description": "This is an animate shadow.  Looking like nothing more than an errant patch of normal shadow, it draws your attention by the way it moves subtly and whispers softly in the back of your mind.  Strange intrusive thoughts accompany the quiet murmur, awakening your most horrific memories and fears.",
+    "description": "An animate shadow, looking like nothing more than an errant patch of lightlessness.  It draws your attention by the way it moves subtly and whispers softly in the back of your mind that speak gently of the things that frighten you most, of the horrors you've not quite forgotten...",
```

Comment:
You seem to have confused the verb form of 'whispers' for the plural noun form here. I recommend changing this bit to read something like '...whispers softly in the back of your mind, speaking gently...'.